account_payment_partner/models/res_partner.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2014 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2014 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    # v8 fields : same without the _id suffix
+    supplier_payment_mode_id = fields.Many2one(
+        "account.payment.mode",
+        string="Supplier Payment Mode",
+        company_dependent=True,
+        domain="[('payment_type', '=', 'outbound')]",
+        help="Select the default payment mode for this supplier.",
+    )
+    customer_payment_mode_id = fields.Many2one(
+        "account.payment.mode",
+        string="Customer Payment Mode",
```

Comment:
string="Customer Payment Mode", could be removed here, as it's the default value